docs/reference/technologies/server/dotnet.mdx

```diff
@@ -69,7 +69,7 @@ dotnet add package OpenFeature
 public async Task Example()
         {
             // Register your feature flag provider
-            Api.Instance.SetProvider(new InMemoryProvider());
+            await Api.Instance.SetProvider(new InMemoryProvider());
```

Comment:
@toddbaert has the functionality been released? If not, we should consider removing it from the readme.